config/oauth_application.yml

```diff
@@ -0,0 +1,4 @@
+oauth_app_name:     'changeme'
+oauth_callback_url: 'http://changeme/callback'
```

Comment:
If you want to use this for seed
we shouldn't set uid and secret, for example model must generate it if empty
I bet in admin panel when we add an app we don't specify it
also we may want to load a localhost test app for env development.

about config/oauth_application.yml
I think we need a better name also we need to be able to list many applications.

Reply:
@mod Valentine asked me for ability to set id and token so we don't need to redeploy peatio after we redeploy barong. This way we can just create an app with secrets that current peatio uses.